lib/app.js

```diff
@@ -22,15 +21,27 @@ const otpConfig = require(YAML_CONFIG)
 // defaults to ./config.js
 const jsConfig = require(JS_CONFIG).configure(otpConfig)
 
-const {getItineraryFooter, LegIcon, ModeIcon} = jsConfig
+const {
+  getItineraryFooter,
+  getContainerSlot,
+  getMapContainerSlot,
```

Comment:
Probably don't need to add the `slot` suffix since the other items don't have this naming convention.

Reply:
I think the slot suffix (or some other suffix) should stay, since these are actually for items to go inside either the map container (but not the map container itself) or the `Row` larger component. This is not the same behavior as `getMainPanel`, which just expects the panel itself.